src/iqm/qiskit_iqm/iqm_backend.py

```diff
@@ -60,6 +61,11 @@ def _dqa_from_static_architecture(sqa: QuantumArchitectureSpecification) -> Dyna
         )
         for gate_name, gate_loci in sqa.operations.items()
     }
+    gates['measure'] = GateInfo(
+        implementations={'__fake': GateImplementationInfo(loci=tuple(tuple([locus]) for locus in qubits))},
+        default_implementation='__fake',
+        override_default_implementation={},
+    )
```

Comment:
Why this special case for `measure`?

Reply:
Ah, that is remaining debug code, I'll remove it